### ISSUE TITLE
[Critical] Select: Custom select does not have a role and/or state

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#586]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87604 |
| **Checkpoint** | 4.1 Compatible › Name, Role, Value (4.1.2.a) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Settings Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/06/26 at 4:37 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The custom select element is missing required roles and/or states or properties.
From Settings menu, select BagIt Profiles, then New, the accept the default AP Trust then select Create. On the General tab, in the Accept BagIt Version if there are multiple items selected/highlighted, that is not announced by the screen reader. See Settings User Experience video at 9:00.
Also occurs on the Manifests tab.

## Recommendation

Rule : 
The name, role, value, states, and properties of user interface components MUST be programmatically determinable by assistive technologies.

How To Fix: 
Fix this issue by using ONE of the following techniques:
1. Use an HTML <select> element with <option> elements to create the drop-down menu.
2. Use the ARIA combobox design pattern. View the full ARIA design pattern for a combobox including expected keyboard interactions and examples: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/

Reference: 
Combobox multiselect: https://cauldron.dequelabs.com/components/Combobox#multiselect
W3C HTML5 Recommendation: https://www.w3.org/TR/html5/sec-forms.html#the-select-element
WAI-ARIA Authoring Practices: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/

Background: 
Every user interface control must have a role to convey what type of control it is for screen reader and other assistive technology users. Native HTML elements - such as <button>, <a>, <input>, <select> - already have a role, so nothing more needs to be done. If you create a custom version of a native HTML element or a custom control or widget that does not have a native HTML equivalent, you must add the relevant role(s) using ARIA as well as expected keyboard interactions.

## Screenshots

![01_multi combobox](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87604/01_multi_combobox.png)
